### ISSUE TITLE
Credentials JWT, auth flow for useGoogleLogin hook

### ISSUE THREAD

**Issue by oobohP** [MomenSherif/react-oauth#348]
Do you think it would be possible to add a optional refs to your GoogleLogin component which flow gives the JWT token

allowing very similar behavior, we can hide the button that is iframed by google, put our own custom button on top and use that when the user clicks our div/button it will give the illusion that our button is the Login with google

this would reduce the overhead needed to call additional apis or exchange auth code's with google on the backend

**Comment by deepakbytebard** [comment]
most awaited component prop

**Comment by jbarop** [comment]
I don't think that is possible because the Login-With-Google button is rendered inside an iframe.

**Comment by jbarop** [comment]
This will not work because the browser prevents access to the iframe's content due to the cross-domain policy.
